plugins/modules/inventory_workflow_manager.py

```diff
@@ -1788,6 +1932,34 @@ def provisioned_wired_device(self):
 
         return self
 
+    # Added function
```

Comment:
What is meant by added function?

Can you write docstring for this API?

Reply:
updated the code